com.cubrid.cubridmigration.core/src/com/cubrid/cubridmigration/core/dbobject/Grant.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (C) 2009 Search Solution Corporation. All rights reserved by Search Solution. 
```

Comment:
For newly added files, use copyright as shown below:

`* Copyright 2016 CUBRID Corporation`

Reply:
I'll fix that. Thank you.